.github/workflows/lint.yml

```
@@ -0,0 +1,30 @@
+on: [push, pull_request]
+
+name: Lint
+
+jobs:
+    php-cs-fixer:
+        name: PHP-CS-Fixer
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@master
+
+            - name: "Run PHP-CS-Fixer"
+              uses: docker://oskarstark/php-cs-fixer-ga:2.16.1
+              with:
+                  args: --ansi --verbose --diff --dry-run
+
+    composer-normalize:
+        name: composer-normalize
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@master
+
+            - name: "Run composer-normalize"
+              uses: docker://localheinz/composer-normalize-action:0.7.0
```

```suggestion
              uses: docker://localheinz/composer-normalize-action:0.5.2
```

Needs a fix - might get around to it next week!